pages/public_cloud/public_cloud_databases/information_02_lifecycle_policy/guide.en-gb.md

```diff
@@ -77,9 +77,9 @@ MySQL EOL will coincide with the official MySQL Lifecycle schedule for linux : <
 
 PostgreSQL EOL will coincide with the official PostgreSQL versioning policy : <https://www.postgresql.org/support/versioning/>
 
-### Caching
+### Caching/Valkey
 
-Caching EOL will coincide with the official Redis® Community Releases policy : <https://redis.io/topics/releases>
+Caching/Valkey EOL will coincide with the official Redis® Community Releases policy : <https://redis.io/topics/releases>
```

Comment:
Create a line for Valkey linking to this address: https://valkey.io/topics/releases/